Formula/mono.rb

```diff
@@ -38,12 +43,13 @@ class Mono < Formula
   link_overwrite "lib/mono"
   link_overwrite "lib/cli"
 
+  # When upgrading Mono, make sure to use the revision from
+  # https://github.com/mono/mono/blob/mono-#{version}/packaging/MacSDK/fsharp.py
   resource "fsharp" do
-    url "https://github.com/dotnet/fsharp.git",
-        tag:      "v11.0.0-beta.20471.5",
-        revision: "03283e07f6bd5717797acb288cf6044cedca2202"
-    # F# patches hhen upgrading Mono, make sure to use the revision from
-    # https://github.com/mono/mono/blob/mono-#{version}/packaging/MacSDK/fsharp.py
+    url "https://github.com/dotnet/fsharp/archive/9cf3dbdf4e83816a8feb2eab0dd48465d130f902.tar.gz"
+    sha256 "f9820d00c6001ac9b0fde29133365719229765d13c89e37eb7d8c62e457c16b6"
```

Comment:
Intel macOS is now failing on fsharp. It built for me locally via Rosetta. Only other change was switching from git to tarball so may revert and retry.
`/private/tmp/mono-20221001-6772-1kuh5uh/mono-6.12.0.182/.brew_home/.nuget/packages/microsoft.build.tasks.git/1.1.0-beta-20206-02/build/Microsoft.Build.Tasks.Git.targets(24,5): error : Unable to locate repository with working directory that contains directory '/private/tmp/mono--fsharp-20221001-6772-1ezhkru/fsharp-9cf3dbdf4e83816a8feb2eab0dd48465d130f902/src/fsharp/FSharp.Core'. [/private/tmp/mono--fsharp-20221001-6772-1ezhkru/fsharp-9cf3dbdf4e83816a8feb2eab0dd48465d130f902/src/fsharp/FSharp.Core/FSharp.Core.fsproj]`

Reply:
Looks like git worked instead of tarball.